dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/frameworks/EuTaxonomyNonFinancials.kt

```diff
@@ -26,7 +26,7 @@ class EuTaxonomyNonFinancials {
             receivedDataMetaInformation.dataType,
         )
         Assertions.assertTrue(
-            listOfDataMetaInfoForTestCompany.contains(receivedDataMetaInformation),
+            listOfDataMetaInfoForTestCompany.component1().dataId.contains(receivedDataMetaInformation.dataId),
             "The all-data-sets-list of the posted company does not contain the posted data set.",
         )
```

Comment:
Why not:
```
        Assertions.assertEquals(
            listOfDataMetaInfoForTestCompany.component1().dataId, receivedDataMetaInformation.dataId,
            "A proper message that acutally tells what is going on here",
        )

```

Reply:
wip

Reply:
updated the message to "The expected data id was not found in the metadata database for the respective company."

Reply:
It's not only about the message, but: 
The current test use in the assertion:
`string.contains(otherString)` (which - in the former implementation - was : `list<string>.contains(otherSTring)`

The current implementation is not needed like this, because we expect full matches. Hence: `assertEquals`

Reply:
Ok, i will have another look at it

Reply:
Full matches do not work because the QA Status of the MetaDataInformation changes between the point where we upload the data and the point where we retrieve them. Directly after uploading the status is "pending" and the retrieved data has status "accepted" due to our setup with the dummy qa service. So i have to think about it how to handle this.

Reply:
Ok, i think i missed the point with the AssertEqual. For some reason i thought we want to check that the whole dataset is equal to the expected one. But i only is applied to the dataId. I change the code. Sorry for the confusion.

Reply:
ok